src/server/lib/okta/registration.ts

```diff
@@ -0,0 +1,18 @@
+import { CreateUserRequestOptions } from '@okta/okta-sdk-nodejs/src/types/models/CreateUserRequest';
+import { oktaClient } from '@/server/lib/okta/client';
+import { UserActivationToken } from '@okta/okta-sdk-nodejs';
+
+export const register = (email: string): Promise<UserActivationToken> => {
```

Comment:
Could this function be called `registerWithOkta` to make it clear [here](https://github.com/guardian/gateway/pull/1131/files#diff-0862d4de5f45856338489930372d58873f6f0cf4ba5423c7b29e04e99d93d131R180) that we are registering with Okta ?

Reply:
👍